main.js

```diff
@@ -0,0 +1,174 @@
+        xMoves.push(move);
+        possMoves.push(possMoves.splice(possMoves.indexOf(move), 1) [0]);
+        possMoves.pop();
+        move.class = 1;
+        turnStatus.textContent = "Player O's turn";
+        console.log(move.class)
+    }
+    
+    //AI random function
+    
+    if(turnStatus.textContent === "Player O's turn"){
+        let randomMove = possMoves[Math.floor(Math.random() * possMoves.length)];
+        randomMove.innerHTML = "<img src= 'https://media2.giphy.com/media/y8Mz1yj13s3kI/giphy.gif' width= 200px height= 200px></img>";    
+        notPossMoves.push(randomMove);
+        xMoves.push(randomMove);
+        possMoves.push(possMoves.splice(possMoves.indexOf(randomMove), 1) [0]);
+        possMoves.pop();
+        randomMove.class = 1;
+        turnStatus.textContent = "Player X's turn";
+    }
+    }
+
+possMoves.forEach( (cells) => {
+    cells.addEventListener('click', playOnePlayer)
+})   
+
+// 2 Player gameplay
+    
+function playTwoPlayer(event){
+
+    //Player X function
+
+    if (possMoves.includes(event.target) && turnStatus.textContent === "Player X's turn"){
+        let move = event.target;
+        move.innerHTML = "<img src= 'https://media1.tenor.com/images/a422941a831e161317fc7d118aa7fac6/tenor.gif?itemid=4957875' height= 200px width= 200px></img>";
+        
+        notPossMoves.push(move);
+        xMoves.push(move);
+        possMoves.push(possMoves.splice(possMoves.indexOf(move), 1) [0]);
+        possMoves.pop();
+        move.class = 1;
+        turnStatus.textContent = "Player O's turn";
+        console.log(move.class)
+    }
+    //Player O function
+    if (possMoves.includes(event.target) && turnStatus.textContent === "Player O's turn"){
+        let move = event.target;
+        move.innerHTML = "<img src= 'https://media2.giphy.com/media/y8Mz1yj13s3kI/giphy.gif' width= 200px height= 200px ></img>";
+        notPossMoves.push(move);
+        oMoves.push(move);
+        possMoves.push(possMoves.splice(possMoves.indexOf(move), 1) [0]);
+        possMoves.pop();
+        move.class = 10;
+        turnStatus.textContent = "Player X's turn"
+    }  
+
+
+console.log(notPossMoves) //debug
+console.log(cellZero)
+
+}
+
+possMoves.forEach( (cells) => {
+    cells.addEventListener('click', playTwoPlayer)
+})
+
+//Winning Function
+
+function win() {
+    
+    winningCombos.forEach(function(solution){
+        let indexZero = solution[0].class;
+        let indexOne = solution[1].class;
+        let indexTwo = solution[2].class
+        if(parseInt(indexZero) + parseInt(indexOne) + parseInt(indexTwo) === 3){
+            turnStatus.textContent ="Player X wins"
+        }
+        else if(parseInt(indexZero) + parseInt(indexOne) + parseInt(indexTwo) === 30){
+            turnStatus.textContent ="Player O wins"
+        
+        } 
+        else {}
+        
+    })
+    onePlayer.addEventListener('click', playOnePlayer)
+    twoPlayer.addEventListener('click', playTwoPlayer)
+
+}
+win()  
```

Comment:
By putting your win function here it will only fire once when the page first loads, and never again. You should probably call it in the event handler for your on click event listeners on the cells.